src/withRootAttribute.js

```diff
@@ -2,21 +2,21 @@ import document from 'global/document';
 import {EVENTS, PARAM_KEY} from './constants';
 import {addons, makeDecorator} from '@storybook/addons';
 
-const updateRootAttribute = ({root, attribute, currentState}) => {
+const updateRootAttribute = ({root = 'html', attribute, currentState}) => {
   const element = (() => {
     if (root === 'body') {
       return document.body;
     }
-    return document.documentElement;
+    if (root === 'html') {
+      return document.documentElement;
+    }
+    return document.querySelector(root) || document.documentElement;
```

Comment:
Maybe we should not fallback to html node, but just skip applying plugin on selected story, which is not have needed element. What do you think?

In this case we can just add after this function:

```js

if (!element) {
  return;
}

```

But maybe you have use case, where it is valid behaviour. What do you think @olya777 ?